airflow/executors/celery_kubernetes_executor.py

```diff
@@ -231,3 +231,7 @@ def send_callback(self, request: CallbackRequest) -> None:
         if not self.callback_sink:
             raise ValueError("Callback sink is not ready.")
         self.callback_sink.send(request)
+
+    @staticmethod
+    def get_cli_commands() -> list:
+        return []
```

Comment:
Hmm, not sure this is right? Don't we want to expose the celery and k8s cli still?

Reply:
This has moved to the celery provider packages since this revision was out. I suppose the right thing here is to expose the celery CLI commands.

Reply:
Actually, I'll include both celery and kube. I think it's safe to assume that dependencies for both should be available even if this code lives within one provider, but it does create a cross provider dependency (but I think by virtue of using this executor the user has accepted that).